frontend/src/components/public_group/PublicGroupExpensesAndActivity.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import PublicGroupDonations from './PublicGroupDonations';
+import PublicGroupExpenses from './PublicGroupExpenses';
+
+export default class PublicGroupExpensesAndActivity extends Component {
```

Comment:
I thought the same at first, however a `{...this.props}` is required and I cannot use it from within a pure functional component